### ISSUE TITLE
[Change]: Authority supervision for sandbox licensure; Part 3, Section 18(2)(b)

### ISSUE THREAD

**Issue by michaelcair0** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#13]
### Affected Legislation

VASP Act (2022) Revision

### Affected Section

Part 3, Section 18(2)(b)

### Proposed Changes

[Authority supervision for sandbox licensure; Part 3, Section 18(2)(b)](https://github.com/MinistryFinancialServices/Virtual-Asset-Amendment-Consultation/blob/b86f85170e238e92fb69a7655bac5e797bbdcf9c/Virtual%20Asset%20(Service%20Providers)%20Act/03%20Part%203%20-%20Sandbox%20Licenses.txt#L24)

Current provision:
>(2) The Authority shall assess, monitor and supervise the innovative service, technology or method of delivery of a sandbox licensee with a view to ensuring that —
(b) the service, technology or method of delivery improves the provision of financial services in the Islands; 

Proposed change (bold & italicized):
(2) The Authority shall assess, monitor and supervise the innovative service, technology or method of delivery of a sandbox licensee with a view to ensuring that —
(b) the service, technology or method of delivery **_is capable of_** improv~~es~~ **_ing_** the provision of financial services in the Islands; 

### Proposal Reasoning

This is a great guidepost. This suggested revision contemplates that regulated financial service providers are generally hesitant to incorporate virtual asset services unless and until they are assured that such services are regulated. The sandbox license should give traditional financial service providers enough of a “green light” to engage with VASPs before an applicant can demonstrate that its services do in fact improve financial
services.

**Comment by Aaron-XReg** [comment]
Agree! I think is could also be revised to "is intended to....". There is no way to guarantee that the service will improve the provision of financial services. The whole point of a sandbox is to experiment with technology that has the potential to improve financial services.
